lib/usd/hdMaya/utils.cpp

```diff
@@ -14,7 +14,7 @@
 // limitations under the License.
 //
 
-#include <hdMaya/utils.h>
```

Comment:
Utils is not a private header file. @HamedSabri-adsk please revert this change.

Reply:
@kxl-adsk Can you clarify what private header mean here?

We have a utils.h and utils.cpp sitting besides each other. Why do I need to do `hdMaya/utils.h` instead of simply `utils.h`

Besides, I have `${CMAKE_CURRENT_SOURCE_DIR}` added to `target_include_directories`  as `PRIVATE` requirement in `lib/usd/hdMaya/CMakeLists.txt`. 

```
target_include_directories(${TARGET_NAME} 
    PRIVATE
        ${CMAKE_BINARY_DIR}/include
        ${CMAKE_CURRENT_SOURCE_DIR}
)
```

Reply:
Hmm... so I was curious if this was / could be made a private header or not - and I think we may need to it to be public, because any hdMaya adapters that others might wish to implement will need/want it.

Currently, it's effectively public, since it's installed.  However, it's referenced in many places using "../hdMaya/utils.h" syntax, which is slightly confusing, and will need to be fixed to get everything in line with our style conventions.

Reply:
> 
> 
> Hmm... so I was curious if this was / could be made a private header or not - and I think we may need to it to be public, because any hdMaya adapters that others might wish to implement will need/want it.
> 
> Currently, it's effectively public, since it's installed. However, it's referenced in many places using "../hdMaya/utils.h" syntax, which is slightly confusing, and will need to be fixed to get everything in line with our style conventions.

Please see https://github.com/Autodesk/maya-usd/pull/369 and https://github.com/Autodesk/maya-usd/pull/370